### PR TITLE
Miscellaneous improvements

### PR DESCRIPTION
**Pull request by gewarren**
- Fix some typeparamref tags
- Improve System.Index docs

*[Hide whitespace changes in diff.](https://github.com/dotnet/dotnet-api-docs/pull/7348/files?diff=unified&w=1)*

### CODE REVIEW

**Review by AaronRobinsonMSFT — APPROVED** [review]
InteropServices &ndash; LGTM.

**Comment by BillWagner** [comment]
closing and reopening for a fresh build.

**Review by BillWagner — APPROVED** [review]
This LGTM @gewarren 

You can :shipit: when ready.

File: xml/System/Index.xml [hunk 1]
```diff
@@ -75,9 +75,10 @@ int lastElement = someArray[^1]; // lastElement = 5
         <Parameter Name="fromEnd" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="value">The index value. It has to be greater or equal than zero.</param>
-        <param name="fromEnd">A boolean indicating if the index is from the start (<see langword="false" />) or from the end (<see langword="true" />) of a collection.</param>
-        <summary>Initializes a new <see cref="T:System.Index" /> with a specified index position and a value that indicates if the index is from the start or the end of a collection.</summary>
+        <param name="value">The index value. It has to be greater then or equal to zero.</param>
+        <param name="fromEnd">
+          <see langword="false" /> if the index is from the beginning of the collection; <see langword="true" /> if the index is from the end of the collection.</param>
```
Comment (gewarren) [inline xml/System/Index.xml:80]:
```suggestion
          <see langword="true" /> to index from the end of the collection, or <see langword="false" /> to index from the beginning of the collection.</param>
```

**Pull request merged**